publication-commons/src/test/java/no/unit/nva/publication/model/storage/DaoUtils.java

```diff
@@ -32,71 +33,72 @@
 import nva.commons.core.attempt.Try;
 
 public final class DaoUtils extends TestDataSource {
-    
+
     public static Resource sampleResource(Publication publication) {
-        var additionalIdentifier = Set.of(new AdditionalIdentifier(CRISTIN_SOURCE, randomString()));
+        var additionalIdentifier = Set.of(
+            (AdditionalIdentifierBase) new AdditionalIdentifier(CRISTIN_SOURCE, randomString()));
```

Comment:
Is this cast a bit weird?

Reply:
yup I agree but the IDE complained
![image](https://github.com/BIBSYSDEV/nva-publication-api/assets/554713/b4c259d5-71bc-49ed-a033-73b9ac72bffa)